client/public/index.html

```diff
@@ -5,26 +5,26 @@
 
 	  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 	  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-
+	  
```

Comment:
Double check your tab settings to be sure it is using two spaces instead of tab characters